eZ/Publish/Core/Helper/PreviewLocationProvider.php

```diff
@@ -76,7 +76,7 @@ public function loadMainLocation($contentId)
                     'status' => Location::STATUS_DRAFT,
                     'parentLocationId' => $parentLocations[0]->id,
                     'depth' => $parentLocations[0]->depth + 1,
-                    'pathString' => $parentLocations[0]->pathString . '/x',
+                    'pathString' => $parentLocations[0]->pathString . 'x/',
```

Comment:
If the Methode returns "/1/2//x" the "Criterion/Subtree" returns a InvalidArgumentException.